res/css/structures/_TabbedView.pcss

```
@@ -164,3 +165,24 @@ limitations under the License.
     overflow: auto;
     min-height: 0; /* firefox */
 }
+
+@media (max-width: 768px) {
```

A comment might be helpful:

```suggestion
/* Hide the labels on tabs, showing only the icons, on narrow viewports. */
@media (max-width: 768px) {
```